pyaedt/application/Design.py

```diff
@@ -549,6 +549,19 @@ def __init__(
         close_on_exit=False,
         student_version=False,
     ):
+        self.oboundary = None
+
+        self.omodelsetup = None
+
+        self.oimportexport = None
+
+        self.ooptimetrics = None
+
+        self.ooutput_variable = None
+
+        self.odesktop = None
+
+        self.oanalysis = None
```

Comment:
@maxcapodi78 Maybe we don't need to have new lines between each data member assignation. Your call.